src/concepts/the-sane-helm-guide.md

```diff
@@ -14,15 +14,15 @@ But Helm does two things well:
 - packaging
 - and templating
 
-In this guide you will get practical knowledge on how-to use Helm focusing on those usecases, packaging and templating, that by the way also underpin Gimlet.
+In this guide, you will get practical knowledge on how to use Helm, focusing on those use-cases, packaging and templating, that, by the way, also underpin Gimlet.
```

Comment:
I've been fighting with this sentence for a while. It also popped up on your radar. Good.

How about we remove the `, by the way,` completely?

Reply:
Thought about this one for a little bit. How about something like this?

"In this guide, you will get practical knowledge about using Helm, focusing on those use-cases, namely packaging and templating, that also underpin Gimlet."